components/shared/embedder/lib.rs

```diff
@@ -438,49 +439,30 @@ pub struct WebResourceRequest {
     )]
     #[ignore_malloc_size_of = "Defined in hyper"]
     pub headers: HeaderMap,
-    pub url: ServoUrl,
+    pub url: Url,
```

Comment:
I'm curious about the rationale for using `url` over `ServoUrl`? Is it for compatibility with broader ecosystem? Is `ServoUrl` ?

Reply:
Yes, that's right. `rust-url` is controlled by the `servo` organization so it makes sense that the API surface uses these types rather than a special servo wrapper which might be confusing to everyone.